src/main/java/com/google/cloud/discotoproto3converter/ConverterApp.java

```diff
@@ -84,6 +87,7 @@ public void convert(String[] args) throws IOException {
     // Optional Parameters
     parsedArgs.put("--service_ignorelist", "");
     parsedArgs.put("--message_ignorelist", "");
+    parsedArgs.put("--enums_as_strings", "false");
```

Comment:
Ditto on default

Reply:
Please see the other comment on default.